src/Domain/CoordinateMaps/KerrHorizonConforming.hpp

```diff
@@ -69,7 +69,10 @@ class KerrHorizonConforming {
   static constexpr size_t dim = 3;
   /*!
    * constructs a Kerr horizon conforming map.
-   * \param spin : the dimensionless spin
+   * \param spin The spin parameter $\vec{a} = \frac{\vec{S}}{M} =
+   * M \vec{\chi}$, where $M$ is the Kerr mass parameter, $\vec{S}$ is the
+   * angular momentum or quasilocal spin, and $\vec{\chi}$ is the
+   * dimensionless spin.
    */
   explicit KerrHorizonConforming(std::array<double, 3> spin);
```

Comment:
(Optional) In an effort to avoid this ambiguity in the future, do you think it's worth it to also rename this variable (and the member variable `spin_`) to something like `dimensionful_spin`? That way both the docs and the code itself are unambiguous?

Reply:
I don't know, I don't particularly like the name "dimensionful spin" so I'm inclined to leave it at "spin".

Reply:
Consensus in the past has been to be explicit.  (See changes in #595, for example.)  This should spell it out, particularly since it makes the opposite choice from the Kerr-Schild solution.